packages/perseus/package.json

```diff
@@ -45,6 +45,7 @@
         "@khanacademy/wonder-blocks-popover": "^3.0.0",
         "@khanacademy/wonder-blocks-progress-spinner": "^2.0.17",
         "@khanacademy/wonder-blocks-spacing": "^3.0.5",
+        "@khanacademy/wonder-blocks-switch": "^1.1.6",
```

Comment:
This dependency should also be added to the `peerDependencies` section.

Reply:
thanks!